sources/packages/backend/apps/db-migrations/src/sql/SFASIndividuals/Add-student-id-sfas-individuals.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE sims.sfas_individuals
+    ADD COLUMN IF NOT EXISTS student_id INTEGER NULL;
```

Comment:
Please add a DB comment to the column also.
I am assuming that there is no integrity check for performance reasons and because it is supposed to be used only during the import part.
If it is true, it can be mentioned in the DB comment also.

Reply:
So the idea of not having the foreign key is for performance right? If that is the case, how costly is the difference?

Not a part of this PR. I believe that nullable columns doesn't need to be explicitly mentioned with NULL(I guess we are doing that in our application). We can ignore the explicit mentioning @ann-aot  @andrewsignori-aot ?

Reply:
@dheepak-aot, yes, the `NULL` does not need to be explicit and we have been just omitting it usually.
About adding the foreign, we can add it.

Reply:
I think I can create the foreign key and if we have any issue with performance we can remove it then.

Reply:
ok. Removed the NULL and added the foreign key. Thanks!